src/iceberg_functions/iceberg_multi_file_list.cpp

```diff
@@ -408,9 +408,9 @@ OpenFileInfo IcebergMultiFileList::GetFileInternal(idx_t file_id, lock_guard<mut
 	const auto &data_file = *found_data_file;
 	const auto &path = data_file.file_path;
 
+	// Unknown data type, could be customized puffin blobs, directly ignore.
 	if (!StringUtil::CIEquals(data_file.file_format, "parquet")) {
-		throw NotImplementedException("File format '%s' not supported, only supports 'parquet' currently",
-		                              data_file.file_format);
+		return OpenFileInfo();
```

Comment:
I don't think this is allowed by the MultiFileList API

It only expects `OpenFileInfo()` at the end of the file list (when file_id >= `TotalFileCount()`)